templates/ObjectSchema.stencil

```diff
@@ -0,0 +1,20 @@
+// --------------------------------------------------------------------------
```

Comment:
This should reference the Header stencil. Would also recommend a name like `Model.stencil` or `StructModel.stencil` (maybe we have `ClassModel.stencil` eventually?).